core/ecschema-editing/src/Editing/Properties.ts

```diff
@@ -0,0 +1,271 @@
+import { ECName, ECObjectsError, ECObjectsStatus, SchemaItemKey, SchemaItemType } from "@itwin/ecschema-metadata";
+import { SchemaContextEditor } from "./Editor";
+import { MutableArrayProperty } from "./Mutable/MutableArrayProperty";
+import { MutableProperty } from "./Mutable/MutableProperty";
+import { MutablePrimitiveOrEnumPropertyBase } from "./Mutable/MutablePrimitiveOrEnumProperty";
+import { MutableClass } from "./Mutable/MutableClass";
+
+type MutablePropertyType = MutableProperty | MutableArrayProperty | MutablePrimitiveOrEnumPropertyBase;
+
+/**
+ * @alpha
+ * A class allowing editing of attributes of the base Property class.
+ */
+export class Properties {
+  public constructor(protected _schemaEditor: SchemaContextEditor) { }
+
+  public async setName(classKey: SchemaItemKey, existingPropertyName: string, newPropertyName: string) {
```

Comment:
For consistency reasons, I'd rename `existingPropertyName` to `propertyName` to match the other signatures.
Also, since it's public, doesn't it require a function comment or isn't that necessary as the class is alpha?

Reply:
done.